modules/phase_field/tests/CahnHilliardFluxBC/isotropic.i

```diff
@@ -0,0 +1,148 @@
+[Mesh]
+  type = GeneratedMesh
+  dim = 2
+  nx = 10
+  ny = 10
+  xmax = 5.0
+  ymax = 5.0
+  elem_type = QUAD4
+[]
+
+[Variables]
+  [./c]
+    order = FIRST
+    family = LAGRANGE
+  [../]
+  [./w]
+    order = FIRST
+    family = LAGRANGE
+  [../]
+  [./d]
+    order = FIRST
+    family = LAGRANGE
+    initial_condition = 1
+  [../]
+[]
+
+[Preconditioning]
+  [./SMP]
+    type = SMP
+    coupled_groups = 'c,w'
+  [../]
+[]
+
+[Kernels]
+  [./cres]
+    type = SplitCHParsed
+    variable = c
+    f_name = F
+    kappa_name = kappa_c
+    w = w
+  [../]
+  [./wres]
+    type = SplitCHWRes
+    variable = w
+    mob_name = M
+  [../]
+  [./time]
+    type = CoupledTimeDerivative
+    variable = w
+    v = c
+  [../]
+
+  [./diff]
+    type = MatDiffusion
+    D_name = 10.0
+    variable = d
+  [../]
+  [./time2]
+    type = TimeDerivative
+    variable = d
+  [../]
+[]
+
+[BCs]
+  [./in_flux]
+    type = CahnHilliardFluxBC
+    variable = w
+    boundary = top
+    flux = '0 0.2 0'
+    mob_name = M
+    args = 'c d'
```

Comment:
Is there a way to not have to specify the args for the mobility twice like this?  Seems a little error prone, since if you change it the material used for the mobility, you have to go fix up all your BCs too...